### PR TITLE
Create BackgroundVideoPlayer component

### LINKED ISSUE #1383: Create a BackgroundVideoPlayer component
Component for background video play without any fancy features of the current `VideoPlayer.` Can be just a simple wrapper for native `<video>`. This should be used in featured video category cards and video hero.

Component name TBD.

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1383

### CODE REVIEW

**Comment by drillprop** [comment]
Draft, needs some tweaks

File: src/components/BackgroundVideoPlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  useEffect(() => {
+    if (!videoRef.current) {
+      return
+    }
+    if (playing) {
+      videoRef.current.play()
+    } else {
+      videoRef.current.pause()
+    }
+  }, [autoPlay, playing])
```
Comment (kdembler) [inline src/components/BackgroundVideoPlayer.tsx]: This hook has a dependency for `autoPlay` but doesn't use it. If this is intentional then let's add a comment with explanation

File: src/components/VideoHero/VideoHero.tsx [hunk 2]
```diff
@@ -103,10 +102,10 @@ export const VideoHero: React.FC = () => {
             <ButtonsContainer>
               <Button
                 size="large"
-                to={absoluteRoutes.viewer.video(coverVideo ? coverVideo.video.id : '')}
-                icon={<SvgActionPlay />}
+                onClick={() => setVideoPlaying((prev) => !prev)}
+                icon={videoPlaying ? <SvgActionPause /> : <SvgActionPlay />}
               >
-                Play
+                {videoPlaying ? 'Pause' : 'Play'}
```
Comment (kdembler) [inline src/components/VideoHero/VideoHero.tsx]: The current behaviour is correct, the goal of the Play button is to navigate to the full video, not control the background playback

File: src/components/BackgroundVideoPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  position: absolute;
+  object-fit: cover;
+  max-height: 100%;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`
+
+export default BackgroundVideoPlayer
```
Comment (kdembler) [inline src/components/BackgroundVideoPlayer.tsx]: Let's use a named export ;)

**Review by kdembler — CHANGES_REQUESTED** [review]

File: src/components/BackgroundVideoPlayer.tsx (hunk 1 above)
Comment (drillprop) [reply src/components/BackgroundVideoPlayer.tsx]: not intentional, fixed

**Review by kdembler — APPROVED** [review]

**Pull request merged**